src/utils/index.js

```diff
@@ -0,0 +1,17 @@
+const convertFileToBase64 = (file) =>
+  new Promise((resolve, reject) => {
+    const reader = new FileReader();
+    reader.onload = () => resolve(reader.result);
+    reader.onerror = reject;
+
+    reader.readAsDataURL(file.rawFile);
+  });
+
+export const formImageDataToBase64 = async (formData) => {
+  if (formData.image) {
+    const { image } = formData;
+    const base64Image = await convertFileToBase64(image);
+    formData.image = base64Image;
+  }
+  return formData;
+};
```

Comment:
What about:

```suggestion
export const formImageDataToBase64 = async (formData) => ({
  ...formData,
  [IMAGE]: formData.image
    ? await convertFileToBase64(formData.image)
    : undefined,
});
```

And import `IMAGE` from `components/Communication/const`